torch/nn/modules/lazy.py

```diff
@@ -89,7 +89,7 @@ class LazyModuleMixin:
     >>> lazy_mlp = LazyMLP()
     >>> # transforms the network's device and dtype
     >>> # NOTE: these transforms can and should be applied after construction and before any 'dry runs'
-    >>> lazy_mlp = mlp.cuda().double()
+    >>> lazy_mlp = lazy_mlp.cuda().double()
```

Comment:
Note that the cuda() and double() ops are done inplace. So you don't need the assignment here, you can simply do
```python
lazy_mlp.cuda().double()
```